formbuilder/config.js

```diff
@@ -152,6 +157,10 @@ export default {
       jsonSchema: {
         type: "number",
         title: "Edit me",
+        description: "",
+        multipleOf: 1,
+        minimum: 1,
+        maximum: 100,
```

Comment:
Because json schema props need to be defined in the initial schema so we can override them with values provided by the field edition form ones.